python/amici/petab_import.py

```diff
@@ -86,7 +86,8 @@ def get_fixed_parameters(
         # remove overridden parameters (`object`-type columns)
         fixed_parameters = [p for p in fixed_parameters
                             if condition_df[p].dtype != 'O'
-                            and sbml_model.getParameter(p) is not None]
+                            and sbml_model.getParameter(p) is not None
+                            and sbml_model.getRuleByVariable(p) is None]
```

Comment:
uuuuh, so what happens with rules here that don't have a RateRule qualifier?

Reply:
This is about whether these parameters should be considered as AMICI-constant-parameters. Any parameters that are rule targets cannot be constants. Or am I missing something?